.github/workflows/doc.yml

```diff
@@ -1,9 +1,11 @@
-name: doc
+name: doc and docker
 
 on:
   push:
     branches:
       - master
+  release:
+    types: [published]
```

Comment:
We are already building a Docker image on release (in the `release.yml` script). It might be better and more consistent to upload the same image on both Docker Hub and GitHub. Also one might not want to push to the "nightly docker track" images for release that might get built from some `vX` branch and not from the master branch. If you want I can add the upload to ghcr.io of "release images" to the `release.yml` script myself.